### PR TITLE
Add info about proxy configuration for pack install

### PR DESCRIPTION
**Pull request by lakshmi-kannan**
Docs for https://github.com/StackStorm/st2/pull/3556

### CODE REVIEW

File: docs/source/packs.rst [hunk 1]
```diff
@@ -235,6 +235,63 @@ than personal access tokens and can be configured on the per-repo basis.
...
+
+    ---
+    https_proxy: "https://proxy.server.io:port"
+    proxy_ca_bundle_path: "/etc/ssl/certs/proxy-ca.pem"
+    no_proxy: localhost,127.0.0.1
+
+Proxy configuration via environment vairables for ``st2actionrunner`` process
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can edit ``/etc/default/st2actionrunner`` or the appropriate init file for your OS to
```
Comment (armab) [inline docs/source/packs.rst]: By convention, `/etc/default/st2actionrunner` for Debian family system and `/etc/sysconfig/st2actionrunner` for RedHat family system.

- `deb` https://github.com/StackStorm/st2-packages/blob/master/packages/st2/debian/st2actionrunner%40.service#L13
- `rpm` https://github.com/StackStorm/st2-packages/blob/master/packages/st2/rpm/st2actionrunner%40.service

File: docs/source/packs.rst [hunk 2]
```diff
@@ -235,6 +235,81 @@ than personal access tokens and can be configured on the per-repo basis.
...
+
+For HTTPS proxy with cert:
+
+.. code-block:: ini
+
+    export https_proxy=http://proxy.server.io:port
+    export proxy_ca_bundle_path=/etc/ssl/certs/proxy-ca.pem
+    export no_proxy=localhost,127.0.0.1,0.0.0.0
+
+When using HTTPS proxy with CA bundle (MITM), you must make sure the proxy CA bundle is an accepted
```
Comment (armab) [inline docs/source/packs.rst]: A note indicating that restarting `st2api` and `st2actionrunner` restart is required would be nice.

File: docs/source/packs.rst [hunk 3]
```diff
@@ -235,6 +235,81 @@ than personal access tokens and can be configured on the per-repo basis.
...
+
+Proxy configuration via environment variables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you are running Ubuntu, you can edit ``/etc/default/st2actionrunner`` and
+``/etc/default/st2api`` files to set the proxy configuration environment variables.
+If you are using RHEL, you can edit ``/etc/sysconfig/st2actionrunner`` and
+``/etc/sysconfig/st2api`` to place the proxy configuration environment variables for the
+``st2actionrunner`` and ``st2api`` processes. |st2| picks up these environment variables
+to let pack installer know about the proxy. The file contents should look as follows:
```
Comment (armab) [inline docs/source/packs.rst]: It would be nice to have a note indicating that restarting `st2api` and `st2actionrunner` services is required.

**Comment by LindsayHill** [comment]
Have made a few changes, and have added redirect for removed reference/proxy file (otherwise it never gets removed, and Google indexes the old file)

**Pull request merged**